tianshou/policy/modelfree/pg.py

```diff
@@ -17,7 +17,8 @@
 from tianshou.policy.base import TLearningRateScheduler
 from tianshou.utils import RunningMeanStd
 
-TDistParams = torch.Tensor | tuple[torch.Tensor]
+# TODO: Is there a better way to define this type? mypy doesn't like Callable[[torch.Tensor, ...], torch.distributions.Distribution]
```

Comment:
I don't think it's possible with a prototype either. This just isn't a well-defined callable type, because some functions accept only one tensor, others more than one. The callable type I had initially seems to work, but MyPy didn't like it, as stated in the comment:

![image](https://github.com/thu-ml/tianshou/assets/1295099/74a964e5-6dee-45d5-aa6f-ab8efe9c1737)